src/languages/en-US/commands/misc.json

```
@@ -40,6 +40,49 @@
+		"If my backstory were to be entirely written, there would be 9 parts, each having around 10 chapters (with the exception of `09 - Epilog`, which has only one) comprised of a variable amount sections proportional to the amount of events in them.",
+		"In Nova, our military practices consist of a VR system which simulates a [Deathmatch](https://en.wikipedia.org/wiki/Deathmatch) [battle royale](https://en.wikipedia.org/wiki/Battle_royale_game) of **200** people in a very large map. Our characters and abilities are mapped to our physical capacity and includes all of our skills for maximum inversion.",
+		"In the story, planets and systems are organized in sectors, Nova is in Sector 00 while Earth is in Sector 0A. Each Sector has an archivist who watches over and alerts nearby units when an enemy approaches with hostile intents.",
+		"Most Novans, including myself, have a vehicle named the same as us, there was no need to distinguish the two as the vehicles are completely fine-tuned to us.",
+		"My eyes look bright violet, but if you look closer, there are trillions of different colors, you can see pretty rainbows in them!",
+		"My favorite sport is [maglev](https://en.wikipedia.org/wiki/Maglev) [hoverboarding](https://en.wikipedia.org/wiki/Hoverboard) and I hold some of the several world records at Nova's competitive tracks.",
+		"My heart is made of a extremely rare mineral which needs to be constructed at [Quark](https://en.wikipedia.org/wiki/Quark) level in a laboratory, and takes around 100 years to complete.",
+		"My husband's name is Crysrel, a character that used to be the story's main character.",
+		"My main planet (Nova) does not use an economy based on money, and {{SHINY}} is actually reactor fuel.",
+		"My military codename is NM-EEA-Y, and it stands for NE-ROM Evangelyn Easton A; Version Y.",
+		"My name comes from a joke a friend made to my creator about my old name (kyraBOT), as I was becoming more complex: **SkyraNET** (based on [SkyNET](https://en.wikipedia.org/wiki/Skynet_%28Terminator%29)).",
+		"My official birthday is on 24th November.",
+		"My skin is made of a very complex nanite mesh with individual solid light projectors, which makes it very similar to a very soft human skin.",
+		"Nova is the universe's biggest artificial planet, it has an approximated radius of 71,500 kilometers. However, both the atmosphere and gravity are very similar to those from Earth, and one day in there equals to 26 hours 12 minutes in Earth.",
+		"Nova's architecture and art is largely inspired by [Solarpunk](https://en.wikipedia.org/wiki/Solarpunk).",
+		"Originally, my entire suit was pearl white and I had mechanical wings. This changed later with blue charcoal with strips of amber, and my wings became visible only when I'm overloading energy.",
+		"The bot's code was rewritten 14 times over the span of 4 years.",
```

```suggestion
		"My code was rewritten 14 times over the span of 4 years.",
```

Something something make Skyra more personal issue